rocAL/rocAL/include/pipeline/node.h

```diff
@@ -24,34 +24,33 @@ THE SOFTWARE.
 #include <set>
 #include <memory>
 #include "graph.h"
-#include "image.h"
+#include "tensor.h"
 #include "meta_data_graph.h"
 class Node
 {
 public:
-    Node(const std::vector<Image *> &inputs, const std::vector<Image *> &outputs) :
+    Node(const std::vector<rocalTensor *> &inputs, const std::vector<rocalTensor *> &outputs) :
         _inputs(inputs),
         _outputs(outputs),
         _batch_size(outputs[0]->info().batch_size()) {}
     virtual ~Node();
     void create(std::shared_ptr<Graph> graph);
     void update_parameters();
-    std::vector<Image*> input() { return _inputs; };
-    std::vector<Image*> output() { return _outputs; };
-    void add_next(const std::shared_ptr<Node>& node) {} // To be implemented
-    void add_previous(const std::shared_ptr<Node>& node) {} //To be implemented
+    std::vector<rocalTensor *> input() { return _inputs; };
+    std::vector<rocalTensor *> output() { return _outputs; };
+    void add_next(const std::shared_ptr<rocalTensor>& node) {} // To be implemented
```

Comment:
Was the change here in L[41] intended ?
void add_previous(const std::shared_ptr<Node>& node) ->  void add_next(const std::shared_ptr<rocalTensor>& node)

Reply:
Will revert back